### PR TITLE
Rocky9

### PR DESCRIPTION
**Pull request by pwalczysko**
- Bump python to 3.9
- Run tests on Rocky9
- Add test on Ubuntu 22.04
- Bump default version of django-prometheus (otherwise test fails with inability to restart omero-web)
- Remove Dockerfile

@jburel @khaledk2

### CODE REVIEW

**Comment by pwalczysko** [comment]
Tests are passing here.

**Review by sbesson — DISMISSED** [review]
The bump to `django-prometheus 2.3.1` is consistent with the release notes of the upstream project - see https://github.com/korfuri/django-prometheus/blob/v2.3.1/CHANGELOG.md

**Review by khaledk2 — DISMISSED** [review]

File: meta/main.yml [hunk 1]
```diff
@@ -4,11 +4,14 @@ galaxy_info:
   description: OMERO.web django-prometheus exporter
   company: Open Microscopy Environment
   license: BSD
-  min_ansible_version: 2.3
+  min_ansible_version: 2.10
```
Comment (jburel) [inline meta/main.yml:7]: do we need that bump?
I think this was needed when declaring dependencies via a requirement file

Comment (pwalczysko) [reply meta/main.yml:7]: > do we need that bump? I think this was needed when declaring dependencies via a requirement file

I will revert it. See https://github.com/ome/ansible-role-ssl-certificate/pull/19#discussion_r1457360568

Comment (pwalczysko) [reply meta/main.yml:7]: fixed in https://github.com/ome/ansible-role-omero-web-django-prometheus/pull/8/commits/55cf85494621cfd6aa4557be1b6924d440bc47e9

**Review by jburel — APPROVED** [review]

**Pull request merged**